frontend/packages/operator-lifecycle-manager/src/components/operand/DEPRECATED_operand-form.tsx

```diff
@@ -805,13 +810,13 @@ export const DEPRECATED_CreateOperandForm: React.FC<OperandFormProps> = ({
           resources={[
             {
               kind: groupVersionKind,
-              namespace: k8sModel.namespaced ? params?.ns : null,
+              namespace: k8sModel.namespaced ? params?.ns : undefined,
             },
           ]}
           desc={displayName}
           placeholder={t('olm~Select {{item}}', { item: kindForReference(groupVersionKind) })}
           onChange={(value) => handleFormDataUpdate(path, value)}
-          selectedKey={currentValue ? `${currentValue}-${k8sModel?.kind}` : null}
+          selectedKey={currentValue ? `${currentValue}-${k8sModel?.kind}` : undefined}
```

Comment:
selectedKey probably accepts a string, so use empty string instead:
```suggestion
          selectedKey={currentValue ? `${currentValue}-${k8sModel?.kind}` : ''}
```